Model/Email.php

```diff
@@ -43,9 +43,9 @@ public function domainEquals(string $domain): bool
      */
     private function getDomain(): ?string
     {
-        if (!$this->domain) {
+        if ($this->domain === null) {
```

Comment:
Actually `$domain` could be left uninitialized with type changed to `?string`. Then, this condition can be replaced with:

`if (!isset($this->domain) {`